Tests/APITesters/AllAPITests/SwiftAPITester/CustomerCenterConfigDataAPI.swift

```diff
@@ -16,12 +16,14 @@ func checkCustomerCenterConfigData(_ data: CustomerCenterConfigData) {
     let localization: CustomerCenterConfigData.Localization = data.localization
     let support: CustomerCenterConfigData.Support = data.support
     let lastPublishedAppVersion = data.lastPublishedAppVersion
+    let productId = data.productId
```

Comment:
Same about these APIs

Reply:
yes, they were in #4226 which I just merged to main